usr/share/escenic/ece-scripts/ece.d/apply-conf.sh

```
@@ -642,6 +642,27 @@ function _apply_conf_read_publication_config() {
 }
 
 
+function _apply_conf_get_additional_resources() {
+  for resourceType in $(compgen -v publication_publication_additional_resource_) ; do
+    # resource = "publication_publication_additional_resource_plugin"
+    local resource=( ${!resourceType} )
+    # resource = ( "/path/to/resource"    "path/to/file"  )
+    if [ ${#resource[@]} == 2 ]; then
+      local fileName=
+      fileName=$(_apply_conf_relativize "${resource[1]}")
+      if [ -f "${fileName}" ]; then
+        echo "${resource[0]}=${fileName}"
+      else
+        _apply_conf_debug "Unable to find file \"${fileName}\" for additional resource \"${resource[0]}\"."
+      fi
+    else
+      # Make the resource fail if it doesn't have exactly two words.
+      _apply_conf_debug "${resource[0]} must have the format \"/resource-path file-name\""
```

>  Make the resource fail

By the look of the code, it's more "silently ignored", isn't it?

---

I believe this was a TODO, I changed the comment.